data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -222,12 +225,38 @@ def _create_tile_data(
             axis=0,
         )
 
+        logger.info("Adding fields for island areas and Puerto Rico")
+        score_tiles[field_names.REGION_FIELD] = np.where(
+            score_tiles[field_names.STATE_FIELD].isin(
+                constants.TILES_PUERTO_RICO_AREAS
+            ),
+            "PR",
```

Comment:
nit: i'm generally averse to acronyms, i feel like the time saved typing is always lost in the time spent trying to read and decipher them. Would you be comfortable expanding these to full names (in whatever case you think is appropriate)? 

second nitty nit: I am unnaturally terrified of strings being used multiple times in code, due to the chance of fat-fingering mistakes when trying to reference the same field by a string name. When using a constant for field names instead, the compiler will discover that the field name is misspecified before the code ever gets committed. I know I'm a little more paranoid than is justified, though. Are you comfortable making constants for these and reusing them?

Reply:
yes to both of these